src/native/libs/System.Security.Cryptography.Native/extra_libs.cmake

```
@@ -4,6 +4,25 @@ macro(append_extra_cryptography_libs NativeLibsExtra)
        set(CMAKE_FIND_LIBRARY_SUFFIXES .a)
     endif(CMAKE_STATIC_LIB_LINK)
 
+    # This is evil, bad, and wrong, but good enough to satisfy the build
```

to please policheck:
```suggestion
    # This is bad and wrong, but good enough to satisfy the build
```